applications/concept-catalogue/src/main/kotlin/no/begrepskatalog/conceptregistration/validation/BegrepValidator.kt

```diff
@@ -0,0 +1,29 @@
+package no.begrepskatalog.conceptregistration.validation
+
+import no.begrepskatalog.generated.model.*
+import org.hibernate.validator.internal.util.ModUtil
+
+fun isValidBegrep(begrep: Begrep): Boolean = when {
+    begrep.status == null -> false
+    begrep.status == Status.UTKAST -> false
+    begrep.anbefaltTerm == null -> false

```

Comment:
Kan potensielt bruke "isNullOrBlank()" for å slå sammen linjene som sjekker null og blank på samme verdi